pkg/virt-handler/vm_test.go

```diff
@@ -1523,7 +1523,8 @@ var _ = Describe("VirtualMachineInstance", func() {
 
 			controller.Execute()
 		}, 3)
-		It("update guest time after completed migration", func() {
+
+		It("should apply post-migration configurations on guest VM after migration completed", func() {
```

Comment:
I would replace `configurations` to `operations`, as we do not limit that things we can do there to config stuff.

Reply:
Done